### ISSUE TITLE
cloud requirement for training YOLO

### ISSUE THREAD

**Issue by BlcaKHat** [pjreddie/darknet#1181]
I contacted a cloud server company for training my data.
how can i reduce my cost for training data-set for a few classes., need some help with it

Our basic plan includes upto 2 projects with 4 cores CPU, 15 GB RAM, and Tesla K80. You get Unlimited CPU Usage with either 39 hours of Spare GPU Time or 10 hours of dedicated GPU Time. This plan costs $10.

If you need more options, we can tailor a plan for you and your data training needs. We will need the following information:

No. of hours of training (note that this point is just referring to the time needed for training, the coding can be done on CPU usage)
Size of data (how much memory for hard disk)
RAM needed (our plan comes with 4 cores CPU and 15 GB RAM, let us know if you can do with less, or require more). Alternatively, you can tell us the batch size for training the model. (Example: I am training 32 images a batch, each image 256x256)
Number of projects for your account

**Comment by jordanlui** [comment]
Best way to reduce your cost of training: train for less epochs.
What company did you train with?

**Comment by BlcaKHat** [comment]
doesn't that affect accuracy ?

**Comment by jordanlui** [comment]
Yes, but it answers your question.
You have provided no information about your training situation, so I gave you the simplest answer.

**Comment by BlcaKHat** [comment]
it's not an answer. you're saying in order to avoid bike accident, use public accident.

**Comment by jordanlui** [comment]
Well I guess that's your opinion. Good luck with your problem solving!

**Comment by BlcaKHat** [comment]
If someone is asking to reduce cost.. that doesn't mean you should stop production.. everyone know that..
